### PR TITLE
Fix readme of product tests to enable MySQL and PostgreSQL connectors

### LINKED ISSUE #5812: [product tests] Default configuration for IntelliJ does not start Presto
Run configuration for Presto in IntelliJ described here: https://github.com/prestodb/presto/blob/master/presto-product-tests/README.md does not start Presto correctly:
```
2016-08-08T14:57:13.320Z	ERROR	main	com.facebook.presto.server.PrestoServer	No factory for connector mysql
java.lang.IllegalArgumentException: No factory for connector mysql
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
	at com.facebook.presto.connector.ConnectorManager.createConnection(ConnectorManager.java:147)
	at com.facebook.presto.metadata.CatalogManager.loadCatalog(CatalogManager.java:99)
	at com.facebook.presto.metadata.CatalogManager.loadCatalogs(CatalogManager.java:77)
	at com.facebook.presto.server.PrestoServer.run(PrestoServer.java:115)
	at com.facebook.presto.server.PrestoServer.main(PrestoServer.java:63)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:147)
```

### PR DESCRIPTION
**Pull request by maciejgrzybek**
This fixes https://github.com/prestodb/presto/issues/5812

### CODE REVIEW

**Comment by maciejgrzybek** [comment]
CC: @sanjay990, @brian-rickman

File: presto-product-tests/conf/presto/etc/catalog/mysql.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
 connector.name=mysql
-connection-url=jdbc:mysql://mysql:3306/test
+connection-url=jdbc:mysql://mysql:13306/test
```
Comment (brian-rickman) [inline presto-product-tests/conf/presto/etc/catalog/mysql.properties:2]: port 13306 only works outside docker.  Travis and run_on_docker.sh both run product tests inside a docker container, so this change will break them.

File: presto-product-tests/conf/presto/etc/catalog/postgresql.properties [hunk 2]
```diff
@@ -1,4 +1,4 @@
 connector.name=postgresql
-connection-url=jdbc:postgresql://postgres:5432/test
+connection-url=jdbc:postgresql://postgres:15432/test
```
Comment (brian-rickman) [inline presto-product-tests/conf/presto/etc/catalog/postgresql.properties:2]: Same issue as for MySQL

File: presto-product-tests/conf/presto/etc/catalog/mysql.properties (hunk 1 above)
Comment (maciejgrzybek) [inline presto-product-tests/conf/presto/etc/catalog/mysql.properties:2]: There are two possibilities how to fix this problem:
1. remove that mapping and follow the same pattern as we do for Hive (1-to-1 mapping).
2. prepare different set of configurations for running from IntelliJ and on automation (possible issues of getting those two sets of configuration out of sync).

Any thoughts?
CC: @arhimondr, @ArturGajowy, @fiedukow

**Comment by maciejgrzybek** [comment]
Changed (in 31a4ea4) port mapping from 13306 to 3306 and 15432 to 5432. Should be consistent for in docker and outside docker runs.

**Comment by maciejgrzybek** [comment]
Changed from default port 5432 to 15432 and 3306 to 13306 to avoid port clash on Travis and dev env. Now direct port mapping is possible (15432:15432 and 13306:13306), so test execution is unified among Travis, Jenkins and dev env.

**Comment by brian-rickman** [comment]
This seems like a much better solution.

**Comment by arhimondr** [comment]
I think we should merge this PR and https://github.com/Teradata/presto/pull/280 and https://github.com/prestodb/presto/pull/5823 (maybe partially) into single one and push it to Facebook/master.

**Comment by maciejgrzybek** [comment]
@arhimondr: Isn't it easier to keep them separate to make review of smaller PRs simpler?

**Comment by maciejgrzybek** [comment]
Started PR against prestodb: https://github.com/prestodb/presto/pull/5833

**Pull request closed without merging**